closed/src/java.base/share/classes/java/io/ClassNameCache.java

```diff
@@ -18,7 +18,7 @@
  *
  * You should have received a copy of the GNU General Public License version
  * 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
- * 
+ *
```

Comment:
Did you intentionally omit this comment present in jdknext?
```
 *  Renamed from ClassCache to avoid name collision with OpenJDK java.io.ClassCache introduced by
 *  JDK-8199589 : ObjectStreamClass does not provide a mechanism to clear the cache.
 *  This class is referred by java.io.ObjectInputStream.
```

Reply:
I didn't notice it. I'll add it if you want, it doesn't seem necessary.

Reply:
I agree it's not necessary, but was hoping it was a conscious decision.
As long as it's in the history (either here or in a commit message), I think it's fine.